tsconfig.json

```diff
@@ -13,9 +13,6 @@
     "outDir": "lib",
     "strict": true,
     "moduleResolution": "node",
-    "types": [
```

Comment:
@mirceanis are these needed? My tests in Webstorm wouldn't execute until I removed it and I've never seen that before.

Reply:
If it causes issues, then it's ok to remove them.

They were added to avoid such issues in the first place